glide.yaml

```diff
@@ -8,61 +8,75 @@ import:
+- package: k8s.io/cloud-provider
+  repo:    https://github.com/openshift/kubernetes-cloud-provider.git
+  version: origin-4.2-kubernetes-1.14.0
+- package: k8s.io/cluster-bootstrap
+  repo:    https://github.com/openshift/kubernetes-cluster-bootstrap.git
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/code-generator
   repo:    https://github.com/openshift/kubernetes-code-generator.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
+- package: k8s.io/component-base
+  repo:    https://github.com/openshift/kubernetes-component-base.git
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/csi-api
   repo:    https://github.com/openshift/kubernetes-csi-api.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
+- package: k8s.io/csi-translation-lib
+  repo:    https://github.com/openshift/kubernetes-csi-translation-lib.git
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kube-aggregator
   repo:    https://github.com/openshift/kubernetes-kube-aggregator.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kube-controller-manager
   repo:    https://github.com/openshift/kubernetes-kube-controller-manager.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kubelet
   repo:    https://github.com/openshift/kubernetes-kubelet.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kube-proxy
   repo:    https://github.com/openshift/kubernetes-kube-proxy.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kube-scheduler
   repo:    https://github.com/openshift/kubernetes-kube-scheduler.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/metrics
   repo:    https://github.com/openshift/kubernetes-metrics.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/sample-apiserver
   repo:    https://github.com/openshift/kubernetes-sample-apiserver.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/sample-cli-plugin
   repo:    https://github.com/openshift/kubernetes-sample-cli-plugin.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/sample-controller
   repo:    https://github.com/openshift/kubernetes-sample-controller.git
-  version: origin-4.1-kubernetes-1.13.4
+  version: origin-4.2-kubernetes-1.14.0
 - package: k8s.io/kubernetes
   repo:    https://github.com/openshift/kubernetes.git
-  version: origin-4.1-kubernetes-1.13.4
- # this matches the 1.13 branch from kube
+  version: origin-4.2-kubernetes-1.14.0
+ # this matches the 1.14 branch from kube
 - package: k8s.io/gengo
   version: 51747d6e00da1fc578d5a333a93bb2abcbce7a95
```

Comment:
no update of gengo? Are we sureß

Reply:
Double checked, no changes in upstream to gengo.

Reply:
I mean in 1.14 branch: https://github.com/openshift/kubernetes/blob/origin-4.2-kubernetes-1.14.0/Godeps/Godeps.json#L4239